installer/templates/phx_web/components/layouts.ex

```diff
@@ -11,6 +11,20 @@ defmodule <%= @web_namespace %>.Layouts do
 
   embed_templates "layouts/*"
 
+  @doc """
```

Comment:
Hey thanks for this, I was trying 1.80.rc-0 candidate and I was a bit confused of the new way we should use the dynamic layouts, because the flash was not working.